### ISSUE TITLE
Feature request: Click to pause/resume

### ISSUE THREAD

**Issue by The-Phi** [linuxmint/xplayer#134]
Not an issue but a request.

On Youtube for example, clicking on the video can stop it, and clicking on it again resume the video.
There's an [addon for VLC](https://github.com/nurupo/vlc-pause-click-plugin) also which implemented it.

I found it personally very handy.
Other people interested?

**Comment by NathanC** [comment]
Has there been any movement on this? I see a related commit above,  but left click doesn't pause/play currently.

**Comment by leigh123linux** [comment]
That commit was never submitted, it's only on a private fork.

**Comment by SergiusTheBest** [comment]
@leigh123linux No, it was submitted here: https://github.com/linuxmint/xplayer/pull/145
